app/src/main/res/layout/item_parent_check_order.xml

```diff
@@ -12,38 +12,52 @@
         android:layout_height="wrap_content"
         android:background="@color/color_dark_slate_gray"
         android:descendantFocusability="blocksDescendants"
-        android:padding="@dimen/dp_15"
         >
-
+        <ImageView
+            android:layout_marginLeft="@dimen/dp_5"
+            android:id="@+id/image_item"
+            android:layout_width="@dimen/dp_60"
+            android:layout_height="@dimen/dp_40"
+            android:layout_alignParentLeft="true"
+            android:layout_alignParentStart="true"
+            android:layout_centerVertical="true"
+            android:background="@android:color/transparent"
+            android:scaleType="fitCenter"
+            android:src="@drawable/ic_new"
+            />
         <TextView
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_alignParentLeft="true"
             android:layout_centerVertical="true"
             android:layout_toLeftOf="@+id/text_date"
+            android:layout_toRightOf="@id/image_item"
             android:text="@{viewModel.userName}"
-            android:textColor="@android:color/white"
-            android:textSize="@dimen/sp_15"
+            android:textColor="@color/color_white"
+            android:textSize="@dimen/sp_16"
             />
         <TextView
             android:id="@+id/text_date"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
+            android:layout_centerVertical="true"
             android:layout_toLeftOf="@id/text_price"
             android:text="@{viewModel.orderDate}"
-            android:textColor="@android:color/white"
-            android:textSize="@dimen/sp_15"
+            android:textColor="@color/color_white"
+            android:textSize="@dimen/sp_16"
             />
 
         <TextView
             android:id="@+id/text_price"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_alignParentRight="true"
+            android:layout_centerVertical="true"
+            android:layout_marginLeft="@dimen/dp_10"
+            android:layout_marginRight="@dimen/dp_5"
```

Comment:
WARNING: [RtlHardcoded] Consider adding `android:layout_marginEnd="@dimen/dp_5"` to better support right-to-left layouts
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. *NOTE*: If your `minSdkVersion` is less than 17, you should add *both* the older left/right attributes *as well as* the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)